src/odemis/driver/pigcs.py

```diff
@@ -3728,14 +3725,14 @@ def sendQueryCommand(self, addr, com):
             assert(len(c) <= 100)  # commands can be quite long (with floats)
 
         if addr is None:
-            full_com = "".join(com)
-            prefix = ""
+            full_com = b"".join(com)
+            prefix = b""
         else:
-            full_com = "".join("%d %s" % (addr, c) for c in com)
-            prefix = "0 %d " % addr
+            full_com = b"".join(b"%d %s" % (addr, c) for c in com)
+            prefix = b"0 %d " % addr
```

Comment:
`com` is a string or a list of string. So you cannot directly get bytes out of them.
I think that the easiest would be to compute first `full_com` as a string (in both sides of the if), then convert it to bytes with `.encode("latin1")`